retry/retry_test.go

```diff
@@ -246,3 +246,30 @@ func TestWaitTime(t *testing.T) {
 		})
 	}
 }
+
+func TestNonEnterpriseConsulError_Error(t *testing.T) {
```

Comment:
Should this be `TestNonRetryableError_Error`?

Reply:
Thanks for the catch!